core/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -429,7 +429,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 IndexTemplateMetaData templateMetaData = cursor.value;
                 builder.startObject(templateMetaData.name());
 
-                builder.field("template", templateMetaData.template());
+                builder.field("template", templateMetaData.patterns());
```

Comment:
[This](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-state.html) is all the docs I could find on this API. So it is fairly vague about the return format.